modules/highgui/src/window_QT.cpp

```diff
@@ -2629,8 +2629,18 @@ void DefaultViewPort::resizeEvent(QResizeEvent* evnt)
 
 void DefaultViewPort::wheelEvent(QWheelEvent* evnt)
 {
-    scaleView(evnt->delta() / 240.0, evnt->pos());
+    int delta = evnt->delta();
+    int cv_event = -1;
+    int flags = ((delta & 0xffff)<<16) | ((evnt->orientation() == Qt::Vertical) ? CV_EVENT_MOUSEWHEEL : CV_EVENT_MOUSEHWHEEL);
+    QPoint pt = evnt->pos();
+
+    icvmouseHandler(evnt, mouse_wheel, cv_event, flags);
```

Comment:
`mouse_wheel` is undeclared identifier. @theg4sh, have you ever tested this code?